example/lib/main.dart

```diff
@@ -506,8 +506,7 @@ class _PullToLoadMorePage extends State<PullToLoadMorePage> {
                   border: OutlineInputBorder(),
                   hintText: 'What are you looking for?'),
             ),
-            pullToLoadMore: true,
-            suggestionsCallback: (pattern, {int? page}) async {
+            suggestionsLoadMoreCallback: (pattern,int? page) async {
```

Comment:
nit: 
```suggestion
            suggestionsLoadMoreCallback: (pattern, page) async {
```

Reply:
As soon as @clragon gives the okay and there aqre no conflicts I'll merge